ironfish-cli/src/commands/status.ts

```diff
@@ -202,6 +203,7 @@ function renderStatus(content: GetNodeStatusResponse, debugOutput: boolean): str
   return `\
 Version              ${content.node.version} @ ${content.node.git}
 Node                 ${nodeStatus}
+Wallet               ${walletEnabled ? 'ENABLED' : 'DISABLED'}  
```

Comment:
There is an `accountStatus` below, which is technically the wallet's chain processor state. Do you think it'd be worth to combine this somehow?

Reply:
Ill think about that one

Reply:
Added the disabled marker to the account status as well

Reply:
I think it's a little odd that we'll have two lines duplicate the same DISABLED information. What if we just update the `accountStatus`? Additionally, we could also rename `Accounts` -> `Wallet`

Reply:
Sweet, that works.